test/extended/networking/multus.go

```diff
@@ -0,0 +1,118 @@
+package networking
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	g "github.com/onsi/ginkgo"
+	o "github.com/onsi/gomega"
+	exutil "github.com/openshift/origin/test/extended/util"
+	"github.com/openshift/origin/test/extended/util/image"
+	kapiv1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	e2e "k8s.io/kubernetes/test/e2e/framework"
+	frameworkpod "k8s.io/kubernetes/test/e2e/framework/pod"
+)
+
+var _ = g.Describe("[sig-network][Feature:Multus-CNI functionality]", func() {
+	var oc *exutil.CLI
+	var ns string // namespace
+
+	oc = exutil.NewCLI("multus-e2e")
+
+	f := oc.KubeFramework()
+	g.It("should ensure that multus-cni functions correctly", func() {
+		makeNamespaceScheduleToAllNodes(f)
+		ns = f.Namespace.Name
+		fmt.Println()
+		o.Expect(testMultus(f, oc, ns)).To(o.Succeed())
```

Comment:
I'd like to keep this as is, if that's okay. There won't be multiple contexts running the testMultus function, but it will eventually look something like this:

```
g.It("should ensure that multus-cni functions correctly", func() {
	ns = f.Namespace.Name
	o.Expect(testMultus(f, oc, ns, valid_annotation)).To(o.Succeed())
	o.Expect(testMultus(f, oc, ns, invalid_annotation)).NotTo(o.Succeed())
	// Another two or three multus tests will live here in the future.
})
```

Or...I'm also open to a "proper" way of setting such a structure up.